console/src/main/scala/io/shiftleft/console/cpgqlserver/CPGQLServer.scala

```diff
@@ -57,7 +57,18 @@ class CPGQLServer(ammonite: EmbeddedAmmonite,
       connection.send(cask.Ws.Text("connected"))
       openConnections += connection
       cask.WsActor {
-        case cask.Ws.Close(_, _) => openConnections -= connection
+        case cask.Ws.ChannelClosed() => {
```

Comment:
doesn't really matter but could merge the two cases `case cask.Ws.ChannelClosed() | cask.Ws.Close(_, _) => {`

Reply:
they do look nicer like that, thx!